lib/travis/api/v3/models/env_var.rb

```diff
@@ -2,7 +2,7 @@ module Travis::API::V3
   class Models::EnvVar < Travis::Settings::Model
     attribute :id, Integer
     attribute :name, String
-    attribute :value, String
+    attribute :value, Travis::Settings::EncryptedValue
```

Comment:
Let's go with `travis-settings` for everything and remove the encryption classes from this repo, as discussed in the call today. 🚀